lib/pharos/phases/configure_client.rb

```diff
@@ -18,19 +18,19 @@ def call
         return if @optional && !kubeconfig?
 
         cluster_context['kubeconfig'] = kubeconfig
-        cluster_context['master-ssh'] = ssh
+        cluster_context['master-ssh'] = host.transport
```

Comment:
```suggestion
        cluster_context['master-transport'] = host.transport
```

Reply:
I dropped the whole thing and replaced with `master_host.transport` where it was used.